Assignment/solution_student/index.html

```diff
@@ -2,7 +2,6 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <title>FEWD HTML Template</title>
     <meta name="description" content="An HTML Starter Template">
     
```

Comment:
Here's where you need to add the code from Google Web Fonts (again, [from here](https://www.google.com/fonts#QuickUsePlace:quickUse/Family:Open+Sans:400,300,300italic,400italic,600,600italic,700italic,700,800,800italic)):
```
<link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,400italic,600,600italic,700italic,700,800,800italic' rel='stylesheet' type='text/css'>
```